graylog2-server/src/main/java/org/graylog/security/entities/EntityDependencyResolver.java

```diff
@@ -66,9 +67,10 @@ public EntityDependencyResolver(ContentPackService contentPackService,
 
     public ImmutableSet<EntityDescriptor> resolve(GRN entity) {
         // TODO: Replace entity excerpt usage with GRNDescriptors once we implemented GRN descriptors for every entity
-        final ImmutableMap<GRN, String> entityExcerpts = contentPackService.listAllEntityExcerpts().stream()
+        final ImmutableMap<GRN, Optional<String>> entityExcerpts = contentPackService.listAllEntityExcerpts().stream()
                 // TODO: Use the GRNRegistry instead of manually building a GRN. Requires all entity types to be in the registry.
-                .collect(ImmutableMap.toImmutableMap(e -> GRNType.create(e.type().name(), e.type().name() + ":").newGRNBuilder().entity(e.id().id()).build(), EntityExcerpt::title));
+                .collect(ImmutableMap.toImmutableMap(e -> GRNType.create(e.type().name(), e.type().name() + ":").newGRNBuilder().entity(e.id().id()).build(),
+                        v -> v.title() != null ? Optional.of(v.title()) : Optional.empty()));
```

Comment:
```suggestion
                        v -> Optional.ofNullable(v.title()));
```
this should work too